### PR TITLE
changes in slides

### PR DESCRIPTION
**Pull request by gxyd**
What changes in am going to make, to make sure no multiple PR's are given?
I will mention the filenames on which i will work (probably)
In this, i will makes changes on `sympy-tutorial/PyCon material/slides`.

### CODE REVIEW

**Comment by aktech** [comment]
The best way to track progress is to create a new folder named PydelhiConf material & add material one by one, PR by PR. Don't change the previous material.

**Comment by gxyd** [comment]
We can go the previous state using `git` in future if we want to mark it as different work.
Or better would be to make it like `release-for-PyDelhi-Conf`? How does that sound?

**Comment by aktech** [comment]
That sounds good, though before making the release I would recommend to create a separate folder, then when we are done we can move the material to main directory and probably delete the SciPy & PyCon folders & then finally make a release.

**Comment by gxyd** [comment]
ping @aktech @leosartaj @shivamvats . I will make other changes in new PR.

File: PyDelhiConf Material/slides/intro.tex [hunk 1]
```diff
@@ -60,7 +60,7 @@
       ``SymPy is an open source Python library for symbolic mathematics. It aims to
       become a full-featured computer algebra system (CAS) while keeping the code as
       simple as possible in order to be comprehensible and easily extensible. SymPy
-      is written entirely in Python and does not require any external libraries.''
+      is written entirely in Python and does not require any non-Python libraries.''
```
Comment (gxyd) [inline PyDelhiConf Material/slides/intro.tex:63]: I just thought to specifically mention this line's change (is it ok?).

Comment (aktech) [inline PyDelhiConf Material/slides/intro.tex:63]: Technically it's right, though the current release, 0.7.6.1, does not require any dependency as mpmath is included. Though we have to explicitly mention it for 1.0, so it's a good idea to mention.

**Comment by aktech** [comment]
Have you generated the slides pdf? I  think the `Makefile` doesn't automatically updates the Authors name in slides. Otherwise It looks good to me.

**Comment by aktech** [comment]
Merging.

**Pull request merged**